plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergPlugin.java

```diff
@@ -123,14 +123,14 @@ public void testRecordingMetastore()
                 .shutdown();
 
         // recording with glue
-        assertThatThrownBy(() -> factory.create(
+        factory.create(
                 "test",
                 Map.of(
                         "iceberg.catalog.type", "glue",
                         "hive.metastore.glue.region", "us-east-2",
                         "hive.metastore-recording-path", "/tmp"),
-                new TestingConnectorContext()))
-                .hasMessageContaining("Configuration property 'hive.metastore-recording-path' was not used");
```

Comment:
That's not an awesome change.
`hive.metastore-recording-path` makes no sense for Iceberg, it will be effectively ignored.

can we do better than adding a TODO?

cc @losipiuk @electrum @sopel39 for guice bindings around Glue Catalog and Glue metastore.

Reply:
> `hive.metastore-recording-path` makes no sense for Iceberg, it will be effectively ignored.

do we have an idea how to address this?

Reply:
I'm fixing it now.